src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+
+            /** @var Meanbee_ConfigPoweredCss_Model_Css $css */
+            $css = Mage::getModel('meanbee_configpoweredcss/css');
+
+            $stores = Mage::app()->getStores();
+            foreach ($stores as $storeId => $store) {
+                try {
+                    Mage::app()->setCurrentStore($storeId);
+                    $css->publish($storeId);
+                } catch (Exception $e) {
+                    fwrite(STDERR, sprintf('There was an error when regenerating the config powered css: %s',
+                        $e->getTraceAsString()));
+                    exit(1);
+                }
+            }
+
+            fwrite(STDOUT,
```

Comment:
Line break formatting is weird here. As I understand it, if you break arguments onto multiple lines then every argument should be on its own line. Final closing brackets will be on their own line too, i.e.
```php
fwrite(
    STDOUT,
    "Your message\n"
);
```